packages/dds/tree/src/test/sequence-change-family/randomSequenceGenerator.ts

```diff
@@ -6,7 +6,12 @@
 import { makeRandom } from "@fluid-internal/stochastic-test-utils";
 import { unreachableCase } from "@fluidframework/common-utils";
 import { AnchorSet, FieldKey, UpPath } from "../../tree";
-import { SequenceEditBuilder, singleTextCursor, Transposed as T } from "../../feature-libraries";
+import { singleTextCursor } from "../../feature-libraries";
+import {
+    SequenceEditBuilder,
+    Transposed as T,
+    // eslint-disable-next-line import/no-internal-modules
```

Comment:
Have you considered adding an eslint configuration for the test directory which disables this rule?

Reply:
I have. I don't really want to encourage people doing this except for the specific files they are testing, but also suppressing it isn't great either, so I don't feel strongly either way.